### ISSUE TITLE
[Bug] built-in screen recorder speeds up the game if above 60 fps

### ISSUE THREAD

**Issue by TeardownDestructionClips** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#603]
### Describe the bug

When recording with the built-in screen recorder the game will get sped up if you are doing above 60fps, the screen recorder is recording at 60fps so anything above that will increase the speed, for example if you are doing 120fps then the game will be twice as fast, if you are doing 90fps then the game will be 50% faster. This can make the game quite frustrating to play at high speeds.

### Steps to reproduce the behavior

1. enable built-in screen recorder mod
2. start recording
3. check if you are doing above 60fps
4. if so, notice that the game is being sped up.

I'm honestly not sure this is easily replicable at all, my entire life I've never been able to achieve above 60fps using the screen recorder so I never even ran into this problem until today, I've seen it happen to one other person a few months ago but this is the first time it's happened to me. I did not update my gpu drivers, change any of my pc components, install new mods, or modify my game files at all, but for some reason my screen recorder is performing twice as good and I'm now experiencing this bug.

### Expected behavior

The game should either lock to 60fps whilst screen recording or the game should play like normal and then discard the excess frames for the 60fps video that is being made.

### Environment

- Teardown Version 1.5.4
- OS: Windows 11
- GPU: RTX 4090
- Graphics driver version: 31.0.15.5222
- CPU: Intel i9-13900k

**Comment by lelanceflamer** [comment]
I think it's because of the makemovie.bat script, it was made for 60 FPS videos and so that's probably why it's accelerated, the game only records your current frame to add it to the capture folder.

**Comment by TeardownDestructionClips** [comment]
This is nothing about the script or the video being made, the actual video that is produced is completely fine and runs at normal speed, the issue is that my game is literally being played at a high speed.

**Comment by TeardownDestructionClips** [comment]
UPDATE: I am no longer achieving that above 60fps anymore, all I did was go to bed and wake up and now it's suddenly back to normal. However, that issues is still there for anyone who manages to get above 60fps, but at least for now I am back to recording at like 40 fps (no destruction)

**Comment by Gregory-Gregory** [comment]
@TeardownDestructionClips hi!

To avoid misunderstanding, if you check the recording itself, will it be sped up? Or does this issue only happen with the gameplay but the recording is normal?

**Comment by TeardownDestructionClips** [comment]
Yes the recording/saved video itself is completely fine, the issue is only noticeable in gameplay

**Comment by nooitaf** [comment]
Reproduce with decent gpu..

1. Set vsync to none
2. Open Simple House Map
3. Start built-in Screen recorder
4. Gameplay is 2x speed :P
5. Recorded frames are normal 60fps

I like this feature of "render out as fast as it can", but i get that it's confusing if you actually have to control something :P
